February 2025 Schedule Update

mayamarshel opened the pull request:
Added update for February 2025 events. Updated calendar. Added GatoradeChalice image to static folder and linked onto page. 'Test pages' were for my understanding of Hugo and have been deleted so should not cause a problem.

mayamarshel committed:
Original Website Version

mayamarshel committed:
Added Test page

mayamarshel committed:
Created test news update

mayamarshel committed:
test push

mayamarshel committed:
test 2

mayamarshel committed:
last update

mayamarshel committed:
deleted pages

mayamarshel committed:
Finished Feburary 2025 news update

- added news blurb about all the nights coming up
- updated  calendar
- did NOT replace image with final image and caption

mayamarshel committed:
Added Gatorade Chalice Image

- added png of image to static folder
- deleted test folder from previous testing
- added caption
- may have to check image view on another device

v2thegreat commented on static/media/image/news/events-february-2025/champagneGatorade.png:
Woah! Who made this?

mayamarshel replied:
'Twas me

v2thegreat commented on static/media/image/news/test/m4m.png:
Why did you remove this?

mayamarshel replied:
I think it was one of the test files I made when playing around on Hugo. I'll double check to make sure its not something important.

cpatulea replied:
It seems this test file was pre-existing in the git repo (since 2023): https://github.com/FOULAB/foulab-dot-org-hugo/blob/production/static/media/image/news/test/m4m.png

I suggest leave this file in the repo as it was.

v2thegreat reviewed (approved):
Great stuff!

mayamarshel committed:
Changed kryma casing

- Changed casing
- reupload to see if calendar is working on main branch

mayamarshel committed:
Updated heading

cpatulea commented on content/news/events-february-2025.md:
Yay I was going to suggest this change, you already got to it!

cpatulea commented on .gitignore:
```diff
@@ -3,3 +3,4 @@ public/*
 resources/*
 *~
 .hugo_build.lock
+content/projects/_index.md
```
I think this should be left out of .gitignore, the file 'content/projects/_index.md' is in the git repo and should remain there.

cpatulea commented on content/news/events-february-2025.md:
```diff
@@ -0,0 +1,61 @@
+headline = "ARTICLE"
+title = "Foulab Events - February 2025"
+lastmod = "2025-01-30 21:52:33-0400"
+++++
```
I think Hugo frontmatter ends with 3 plusses: https://gohugo.io/content-management/front-matter/

You had it right at first.

cpatulea commented on content/news/events-february-2025.md:
```diff
@@ -0,0 +1,61 @@
+lastmod = "2025-01-30 21:52:33-0400"
+++++
+
+{{< centered-image src="/events-february-2025/champagneGatorade.png" >}}
```
Image references should be with a simple filename, no path. Example: https://github.com/FOULAB/foulab-dot-org-hugo/blob/91dfe26930ac1f56b10b645a4979d480f941f647/content/news/events-january-2025.md?plain=1#L7

cpatulea reviewed (changes requested):
First pass looks great! Thanks again for working on this.

mayamarshel committed:
Revert "Added Gatorade Chalice Image"

This reverts commit e057d2f031b3f0022108d1b9fc9eeec39ad9c08d.

mayamarshel committed:
Revert "deleted pages"

This reverts commit b0b0563ced86ee0f93b667d5ff472292556ef161.

mayamarshel committed:
Polished Version With Eigma Changes

- Added back the m4m.png image
- Took index file out of Git Ignore
- Fixed markdown error +++
- Fixed image path name

mayamarshel committed:
Fixed Reverted Edits

- Grammar and spelling in website post
- No RSVP for Warhammer night
